CHANGELOG.md

```diff
@@ -4,17 +4,45 @@ aws-parallelcluster-cookbook CHANGELOG
 This file is used to list changes made in each version of the AWS ParallelCluster cookbook.
 
 
-X.X.X
+2.8.0
 -----
 
 **ENHANCEMENTS**
 
 - Enable support for ARM instances on Ubuntu 18.04 and Amazon Linux 2.
+- Install  PMIx v3.1.5 and provide slurm support for it on all supported operating systems except for
+  CentOS 6.
+- Install glibc-static, which is required to suport certain options for the Intel MPI compiler.
 
 **CHANGES**
 
 - Disable libvirtd service on Centos 7. Virtual bridge interfaces are incorrectly detected by Open MPI and
   cause MPI applications to hang, see https://www.open-mpi.org/faq/?category=tcp#tcp-selection for details 
+- Use CINC instead of Chef for provisioning instances. See https://cinc.sh/about/ for details.
+- Retry when mounting an NFS mount fails.
+- Install the `pyenv` virtual environments used by ParallelCluster cookbook and node daemon code under
+  /opt/parallelcluster instead of under /usr/local.
+- Avoid downloading the source for `env2` at installation time.
+- Drop dependency on Ridley.
+- Vendor cookbooks as part of instance provisioning, rather than doing so before copying the cookbook into an
+  instance. Users no longer need to have `berks` installed locally.
+- Drop the dependencies on the poise-python, tar, adn hostname third-party cookbooks.
+- Use the official CentOS AMIs as the base images for ParallelCluster AMIs.
```

Comment:
We were already using the official one, that was updated. Probably better
"Use the new official CentOS 7 AMI as the base images for ParallelCluster AMI."

Reply:
Done